lite-variables.tf

```diff
@@ -0,0 +1,224 @@
+# All 'aks_' variables are before any other variables.
+variable "aks_4nodepools" {
+  description = "If true 4 nodepools are used. If false 2 are used."
+  type        = bool
+  default     = false
+}
+
+variable "aks_logging_monitoring_enabled" {
+  description = "Used to get logging and monitoring of kubernetes and hpcc cluster."
+  type        = bool
+  default     = false
+}
+
+variable "aks_admin_email" {
+  type        = string
+  description = "REQUIRED.  Email address of the administrator of this HPCC Systems cluster.\nExample entry: jane.doe@hpccsystems.com"
+  validation {
+    condition     = length(regexall("^[^@]+@[^@]+$", var.aks_admin_email)) > 0
+    error_message = "Value must at least look like a valid email address."
+  }
+}
+
+variable "aks_admin_name" {
+  type        = string
+  description = "REQUIRED.  Name of the administrator of this HPCC Systems cluster.\nExample entry: Jane Doe"
+}
+
+variable "aks_azure_region" {
+  type        = string
+  description = "REQUIRED.  The Azure region abbreviation in which to create these resources.\nMust be one of [\"eastus\", \"eastus2\", \"centralus\"].\nExample entry: eastus2"
+  validation {
```

Comment:
I think this is noted elsewhere, but: We should remove the region constraints.  What is there now, which was brought over from old code, pertains to our group and not to OSS.  Users should be free to place their clusters anywhere in the world.